plugins/nanospell/plugin.js

```diff
@@ -358,37 +355,26 @@
 			}
 
 			function checkNow(rootElement) {
-				if (!selectionCollapsed() || self._spellCheckInProgress) {
+				rootElement = rootElement || editor.editable();
+
+				if (!selectionCollapsed() || spellCheckInProgress(rootElement)) {
```

Comment:
Much nicer!

I'm wondering if `triggerSpelling` should be replaced with this `checkNow` method,

they do really similar things, but this seems to be doing more checks now.  Maybe not good to roll into this PR.